assets/js/tabs.js

```diff
@@ -15,19 +15,24 @@
   var interval = 3000;
   var tabCarousel = setInterval(nextSlide, interval);
 
-  $('.tab-content.tab-focus, .nav-tabs.nav-tabs-focus').hover(function(e){
+  $(document).on({
```

Comment:
Oh, I didn't see the target `".tab-content.tab-focus, .nav-tabs.nav-tabs-focus"` in the diff, sorry !